pkg/debug/ssh/sshd_config

```diff
@@ -0,0 +1,5 @@
+AuthorizedKeysFile .ssh/authorized_keys
+AllowTcpForwarding yes
```

Comment:
Alpine's patch: https://git.alpinelinux.org/aports/tree/main/openssh/disable-forwarding-by-default.patch
is disabling the TCP Forwarding by default, which means this change will change our current configuration.

Reply:
Ok, thanks. So the behavior in this case will not change.

Reply:
```
[debug] root@linuxkit-525400123456:/$ cat /etc/ssh/sshd_config | grep '^[^# ]'
AuthorizedKeysFile	.ssh/authorized_keys
AllowTcpForwarding yes
GatewayPorts no
X11Forwarding no
Subsystem	sftp	/usr/lib/ssh/sftp-server
```